src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/validation/FlowValidatorTest.java

```diff
@@ -199,6 +201,18 @@ public void shouldFailOnSpecifiedBothOuterVlansAndVlanStatistics() throws Invali
         flowValidator.checkFlowForCorrectOuterVlansWithVlanStatistics(flow);
     }
 
+    @Test(expected = InvalidFlowException.class)
+    public void shouldFailOnTooLowVlanStatistics() throws InvalidFlowException {
+        RequestedFlow flow = buildFlow(VLAN_1, VLAN_2, newHashSet(STATS_VLAN_TOO_LOW));
+        flowValidator.checkFlowForVlanStatisticsInCorrectRange(flow);
+    }
+
+    @Test(expected = InvalidFlowException.class)
+    public void shouldFailOnTooHighVlanStatistics() throws InvalidFlowException {
+        RequestedFlow flow = buildFlow(VLAN_1, VLAN_2, newHashSet(STATS_VLAN_TOO_HIGH));
+        flowValidator.checkFlowForVlanStatisticsInCorrectRange(flow);
+    }
+
```

Comment:
Also I think i worth to add tests for case `if (vlanStatistics == null || vlanStatistics.isEmpty())`

Reply:
I'd say those are already there
https://github.com/telstra/open-kilda/blob/0f1b8f9c93bcf9ca60d5197f5b96254a8179c797/src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/validation/FlowValidatorTest.java#L184-L194

Reply:
added for both